src/main/java/carpet/settings/SettingsManager.java

```diff
@@ -20,23 +21,9 @@
 import org.apache.commons.lang3.tuple.Pair;
 import org.apache.logging.log4j.util.TriConsumer;
 
-import java.io.BufferedReader;
-import java.io.File;
-import java.io.FileNotFoundException;
-import java.io.FileReader;
-import java.io.FileWriter;
-import java.io.IOException;
-import java.io.PrintStream;
+import java.io.*;
```

Comment:
Please expand these, this is very useful for vc and control of minecraft updates.

Reply:
what is vc? Isn't it version control?

Reply:
yes